skia/RSkComponentProviderBVLinearGradient.cpp

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
Can we change this provider to macro based . Assuming the changes are made in ReactNativeSkia branch
as RNS_SETUP_COMPOMNENT_PROVIDER